src/main/resources/db/migration/postgres/V2__Add_accounts.sql

```diff
@@ -0,0 +1,2 @@
+Insert into accounts(username,password) values ('livepool','18942020');
```

Comment:
PR1: You should encrypt your password in database.

Reply:
@antonyhot I try to use ENCODE to encrypt the password.So may use DECODE to decoding the password.